apps/rush-lib/src/data/test/jsonFiles/rushWithLockVersion.json

```diff
@@ -0,0 +1,18 @@
+{
+  "npmVersion": "4.5.0",
+  "rushMinimumVersion": "2.5.0",
+  "nodeSupportedVersionRange": ">=6.9.0 <7.0.0",
+  "projectFolderMinDepth": 1,
+  "projectFolderMaxDepth": 99,
+
+  "versionPolicies": [
+    {
+      "policyName": "testPolicy1",
+      "baseType": "lockStepVersion",
+      "version": "1.1.0",
```

Comment:
>"version": "1.1.0", [](start = 6, length = 19)

Are you planning to store the lockstep version in rush.json?  That would require reading and rewriting rush.json every time we publish, which is unprecedented.  Normally we store rewritten data in small JSON files in common/config/rush.